### ISSUE TITLE
Lensing task

### ISSUE THREAD

**Issue by Justinezgh** [Justinezgh/SBI-Diff-Simulator#25]
I compress using use loss and here are the results: 

`fiducial_model = condition(model, {'omega_c': 0.3, 'sigma_8': 0.8})`
`sample_map_fiducial = seed(fiducial_model, jax.random.PRNGKey(3))`
`m_data = sample_map_fiducial()`
![image](https://user-images.githubusercontent.com/72011736/192488071-ae500213-f7a8-4ed3-a598-ed67414dc1b8.png)
![image](https://user-images.githubusercontent.com/72011736/192488082-f70ffc94-57c3-4cc1-926a-18e9068abfbe.png)

`fiducial_model = condition(model, {'omega_c': 0.3, 'sigma_8': 0.8})`
`sample_map_fiducial = seed(fiducial_model, jax.random.PRNGKey(28))`
`m_data = sample_map_fiducial()`
![image](https://user-images.githubusercontent.com/72011736/192489196-e0374074-d359-4030-86f0-d2a66c67df96.png)
![image](https://user-images.githubusercontent.com/72011736/192489219-d6297ac9-aba5-4f2c-beba-cd5083299ec5.png)

`fiducial_model = condition(model, {'omega_c': 0.3, 'sigma_8': 0.8})`
`sample_map_fiducial = seed(fiducial_model, jax.random.PRNGKey(30))`
`m_data = sample_map_fiducial()`
![image](https://user-images.githubusercontent.com/72011736/192488605-ccb92b05-3c4d-4f95-9391-fb5f23d2aab5.png)
![image](https://user-images.githubusercontent.com/72011736/192488639-84773ed2-b310-439d-bda9-be47dc3f5d95.png)

`fiducial_model = condition(model, {'omega_c': 0.3, 'sigma_8': 0.8})`
`sample_map_fiducial = seed(fiducial_model, jax.random.PRNGKey(20))`
`m_data = sample_map_fiducial()`
![image](https://user-images.githubusercontent.com/72011736/192488869-296ffd89-5cf5-45f2-838f-b3f5f934583e.png)
![image](https://user-images.githubusercontent.com/72011736/192488903-02ce3a93-2647-447c-b221-d43126d16c1f.png)

Ok so first all these posteriors shouldn't be too far from 0.3 0.8 right ? 
Second, I thought that compression couldn't bias the result so I'm a bit confused ^^'

[notebook](https://github.com/Justinezgh/SBI-Diff-Simulator/blob/u/Justinezgh/lensing/notebooks/dev/LensingForwardModelLogNormal_NPE.ipynb)

**Comment by Justinezgh** [comment]
I'll try with mutual information

**Comment by Justinezgh** [comment]
Hey @dlanzieri :) did you have this kind of results when you were working on the different types of compression ?

**Comment by Justinezgh** [comment]
I have the same behavior with vmim

**Comment by dlanzieri** [comment]
> Hey @dlanzieri :) did you have this kind of results when you were working on the different types of compression ?

Hey @Justinezgh! What kind of simulation are you implementing? I didn't find any difference between the two types of compression with the Lognormal field. I'm now working to see what happens when implementing the N-body simulation.
